src/llmcompressor/modifiers/utils/pytorch_helpers.py

```diff
@@ -100,3 +105,28 @@ def run_calibration_forward(
         torch.cuda.empty_cache()
 
     return intermediates
+
+
+def is_moe_model(model: Module) -> bool:
+    """
+    Check if the model is a mixture of experts model
+
+    :param model: the model to check
+    :return: True if the model is a mixture of experts model
+    """
+
+    # Check for MoE components
+    for _, module in model.named_modules():
+        module_name = module.__class__.__name__
+        if "MoE" in module_name or "Expert" in module_name:
```

Comment:
I've left this if condition as is because it's checking the class name(s), (which should respect the case); even if this condition fails, the one below it should catch if a module is an moe